pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/AstImplUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.ast;
+
+import java.util.Arrays;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import net.sourceforge.pmd.Rule;
+import net.sourceforge.pmd.lang.ast.Node;
+import net.sourceforge.pmd.lang.java.typeresolution.TypeHelper;
+
+/**
+ * KEEP PRIVATE
+ * @author Clément Fournier
+ */
+final class AstImplUtil {
+
+    private static final List<String> UNUSED_RULES
+        = Arrays.asList("UnusedPrivateField", "UnusedLocalVariable", "UnusedPrivateMethod", "UnusedFormalParameter");
+
+    private static final List<String> SERIAL_RULES = Arrays.asList("BeanMembersShouldSerialize", "MissingSerialVersionUID");
+
+    private AstImplUtil() {
+
+    }
+
+    private static boolean isSuppressWarnings(ASTAnnotation astAnnotation) {
+        return TypeHelper.isA(astAnnotation, "java.lang.SuppressWarnings");
```

Comment:
```suggestion
        return TypeHelper.isA(astAnnotation, SuppressWarnings.class);
```

As the class is part of the JRE, we can use the class directly